src/ui/zcl_abapgit_gui_router.clas.abap

```diff
@@ -141,7 +141,7 @@ ENDCLASS.
 
 
 
-CLASS zcl_abapgit_gui_router IMPLEMENTATION.
+CLASS ZCL_ABAPGIT_GUI_ROUTER IMPLEMENTATION.
```

Comment:
Not sure why this is happening. It is lowercase in my branch: https://github.com/EduardoCopat/abapGit/blob/4120-dev/src/ui/zcl_abapgit_gui_router.clas.abap

Reply:
One of many issues https://github.com/abapGit/abapGit/issues/3741. Just ignore this one.